src/Razor/src/rzls/rzls.csproj

```diff
@@ -67,6 +67,6 @@
     </ItemGroup>
 
     <MSBuild Projects="@(ProjectToPublish)" Targets="Publish;_IncludeOmniSharpPlugin" BuildInParallel="false" />
-    <MSBuild Projects="@(ProjectToPublish_PlatformAgnostic)" Targets="Publish;_IncludeOmniSharpPlugin" BuildInParallel="false" />
+    <MSBuild Projects="@(ProjectToPublish_PlatformAgnostic)" Targets="Publish;_IncludeOmniSharpPlugin" BuildInParallel="false" Condition="$([MSBuild]::IsOSPlatform('Windows'))" />
```

Comment:
Thought it was interesting we couldn't just check the `OS` property like we do elsewhere, what's different about this one?

Reply:
We probably can! Might be better for readability actually, I just updated it. We used `$([MSBuild]::IsOSPlatform(...)` earlier in the same file so I originally just re-used it for convenience, but it's really only needed to distinguish between Mac/Linux.